recipes/cutadapt/meta.yaml

```diff
@@ -1,17 +1,19 @@
-{% set version = "4.6" %}
+{% set version = "4.7" %}
 
 package:
   name: cutadapt
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/35/b4/f1a7401c3503c17998fb9547b04353217a18323d5d85a3b957f1049ab800/cutadapt-4.6.tar.gz
-  sha256: 924116f34569248035b16f58e73458ed4c0004e44823b80b07f4ab419272f591
+  url: https://files.pythonhosted.org/packages/92/0f/acf2c1055b31c0cf9edc4a46dd6b900905989d78602aef62293b52057fcb/cutadapt-4.7.tar.gz
+  sha256: 8738a35b363eaf615665a4e7d1b4beb385cd93fb7ffdcf82cd4ab6457acc879b
 
 build:
-  number: 1
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-  skip: True  # [py27 or py36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv"
+  skip: True  # [py < 38]
+  entry_points:
+    - cutadapt = cutadapt.cli:main_cli
```

Comment:
Do you know what the difference is when this setting is added in the recipe? I have always felt that it is redundant to specify entry_points in the recipe because it has always worked without. It also would be annoying to change the entry point in Cutadapt if I decided to do so.